main/EDPftvarcon.F90

```diff
@@ -48,9 +48,11 @@ module EDPftvarcon
      real(r8), allocatable :: crown(:)               ! fraction of the height of the plant 
                                                      ! that is occupied by crown. For fire model. 
      real(r8), allocatable :: bark_scaler(:)         ! scaler from dbh to bark thickness. For fire model.
-     real(r8), allocatable :: crown_kill(:)          ! scaler on fire death. For fire model. 
+     real(r8), allocatable :: crown_resist(:)        ! resistance to fire. (1 = none) For fire model.
+     real(r8), allocatable :: crown_fire(:)          ! Does plant have passive crown fire? (1=yes, 0=no)
```

Comment:
Perhaps change "have" to "susceptible"?

Reply:
also, maybe this parameter should have the name passive in it?

Reply:
updated name and comments